oak_functions/loader/Cargo.toml

```diff
@@ -46,8 +37,8 @@ oak_functions_abi = { path = "../abi" }
 oak_functions_extension = { path = "../extension" }
 oak_functions_lookup = { path = "../lookup" }
 oak_functions_util = { path = "../util" }
-oak_functions_metrics = { optional = true, path = "../experimental/metrics" }
-oak_functions_tf_inference = { optional = true, path = "../experimental/tf_inference" }
+oak_functions_metrics = { path = "../experimental/metrics" }
+oak_functions_tf_inference = { path = "../experimental/tf_inference" }
```

Comment:
Just to confirm: the goal is for the oak_functions_loader crate to *never* depend on these crates (not to *always* depend on them).